docker-compose.yml

```diff
@@ -48,32 +50,6 @@ services:
     image: redis
   mongo:
     image: mongo
-  kobocat:
-    build: ./kobocat
-    depends_on: *default-depends_on
-    environment:
-      <<: *default-environment
-      DATABASE_URL: postgis://postgres:postgres@postgres:5432/kobocat
-    volumes:
-      - ./kobocat:/srv/src/kobocat
-      - ./.vols/kobocat_media_uploads:/srv/src/kobocat/media
-    command: ./manage.py runserver 0.0.0.0:8001
-    ports:
-      - "8001:8001"
-    networks:
-      default:
-        aliases:
-          - kc.kobo.local
-
-  kobocat_worker:
```

Comment:
In kobo compose, all queues run on one worker (gotta go fast) so removing this is sufficient.